src/components/Logout/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { EXPRESS_OAUTH_LOGOUT_URL } from '../../configs/env';
+/** interface to describe props for Logout component
+ * @member {string} logoutURL the url of the logout endpoint of the Oauth server.
+ */
+export interface LogoutProps {
+  logoutURL: string;
+}
+/**
+ * Open another window and navigate to the logout URL.
+ * @param {string} logoutURL URL string representing the auth server logout URL endpoint.
+ * This function takes the approach of opening a new window and navigating to the logout
+ * url of the authentication server in order to go around the browser's CORS policy.
+ */
+export function logoutFromAuthServer(logoutURL: string) {
+  const logoutWindow: Window | null = window.open(logoutURL);
+  const timer: NodeJS.Timeout = setInterval(() => {
+    if (logoutWindow) {
+      logoutWindow.close();
+      window.location.href = EXPRESS_OAUTH_LOGOUT_URL;
```

Comment:
would also be cool, to add a test for this line, I think we already have an example [here](https://github.com/onaio/reveal-frontend/blob/619a65e2b22742eabeb3481c7bff69ba685c92cf/src/components/page/CustomCallback/tests/index.test.tsx#L70)

Reply:
@p-netm due to time constraints I added an issue to attend to this issue later https://github.com/onaio/reveal-frontend/issues/808